indy_node/server/config_req_handler.py

```diff
@@ -70,7 +86,23 @@ def validate(self, req: Request):
                 req.reqId,
                 "Nym {} not added to the ledger yet".format(origin))
         if typ == POOL_UPGRADE:
-            currentVersion = Upgrader.getVersion()
+            pkt_to_upgrade = req.operation.get(PACKAGE, getConfig().UPGRADE_ENTRY)
+            if pkt_to_upgrade:
+                currentVersion, cur_deps = self.curr_pkt_info(pkt_to_upgrade)
```

Comment:
`curr_pkt_info` returns a version only, doesn't it?

Reply:
version and list of dependencies